contracts/test/ERC20toCW20PointerTest.js

```diff
@@ -109,16 +110,33 @@ describe("ERC20 to CW20 Pointer", function () {
                     };
                     // send via eth_ endpoint - synthetic event doesn't show up
                     const ethlogs = await ethers.provider.send('eth_getLogs', [filter]);
-                    expect(ethlogs.length).to.equal(0);
+                    expect(ethlogs.length).to.equal(1);
+                    expect(ethlogs[0]["address"].toLowerCase()).to.equal((await pointer.getAddress()).toLowerCase());
+                    expect(ethlogs[0]["topics"][0]).to.equal(ethers.id("Transfer(address,address,uint256)"));
+                    expect(ethlogs[0]["topics"][1].substring(26)).to.equal(sender.evmAddress.substring(2).toLowerCase());
+                    expect(ethlogs[0]["topics"][2].substring(26)).to.equal(recipient.evmAddress.substring(2).toLowerCase());
 
                     // send via sei_ endpoint - synthetic event shows up
                     const seilogs = await ethers.provider.send('sei_getLogs', [filter]);
                     expect(seilogs.length).to.equal(1);
                     expect(seilogs.length).to.equal(1);
-                    expect(seilogs[0]["address"].toLowerCase()).to.equal((await pointer.getAddress()).toLowerCase());
-                    expect(seilogs[0]["topics"][0]).to.equal(ethers.id("Transfer(address,address,uint256)"));
-                    expect(seilogs[0]["topics"][1].substring(26)).to.equal(sender.evmAddress.substring(2).toLowerCase());
-                    expect(seilogs[0]["topics"][2].substring(26)).to.equal(recipient.evmAddress.substring(2).toLowerCase());
+
+                    // check eth_getBlockByNumber vs sei_getBlockByNumber
+                    const ethBlock = await ethers.provider.send('eth_getBlockByNumber', ['0x' + blockNumber.toString(16), false]);
+                    const seiBlock = await ethers.provider.send('sei_getBlockByNumber', ['0x' + blockNumber.toString(16), false]);
+                    expect(ethBlock.transactions.length).to.equal(1);
+                    expect(seiBlock.transactions.length).to.equal(1);
+
+                    // check eth_getBlockByReceipts vs sei_getBlockByReceipts
+                    const ethReceipts = await ethers.provider.send('eth_getBlockReceipts', ['0x' + blockNumber.toString(16)]);
+                    const seiReceipts = await ethers.provider.send('sei_getBlockReceipts', ['0x' + blockNumber.toString(16)]);
+                    expect(ethReceipts.length).to.equal(1);
+                    expect(seiReceipts.length).to.equal(1);
```

Comment:
do we do any verification on the seiReceipt?

Reply:
checking that the 2 tx hashes are the same between ethReceipts and seiReceipts